doc/manual/pitfalls.rst

```diff
@@ -0,0 +1,27 @@
+.. _pitfalls:
+
+.. contents:: Contents
+   :local:
+   :depth: 2
+
+Pitfalls
+########
+
+Evoked Arithmetic
+=================
+
+Two evoked objects can be contrasted using::
+
+	>>> evoked = evoked_cond1 - evoked_cond2
+
+Note, however that the number of trials used to obtain the averages for ``evoked_cond1`` and ``evoked_cond2`` are taken into account when computing ``evoked``. That is, what you get is a weighted average, not a simple element-by-element subtraction.
```

Comment:
Should mention that `combine_evoked` can be used to use uniform or custom weights.